### DISCUSSION TITLE
Error after upgrading to v4.4.9.0 - avio_open:Protocol not found

### DISCUSSION THREAD

**Discussion by tachappy1** [doitandbedone/ispyagentdvr-docker#560] [General]
After upgrading the Docker image to v4.4.9.0 I get the following error when it attempts to make a recording then it immediately shuts itself down.

I also tried reinstalling the image from scratch with the same result. 

I tried setting the cameras to record from the live stream instead of raw but no change. 

I am running this on a QNAP NAS using Container Station. Have not had any problems until this update. 

Error: Push blocked (Send:False, Enabled:True, Connected: False, Messaging:True)
StartSaving: Float: 
RawWriter: Float: Using stream timestamps for this recording
Error: AVIO Open: AVIO Open: rawwriter: avio_open:Protocol not found
Error: AVIO Open:    at CoreLogic.RealTime.FFmpegBase.Throw(String method, Int32 code)
   at CoreLogic.RealTime.RawWriter.Open(Boolean isRecursive)
**rawwriter: avio_open:Protocol not found**

**Comment by ispysoftware** [comment]
Can you check the filename settings, sounds like you might have a : in the filename field or something like that
https://unix.stackexchange.com/questions/412519/ffmpeg-protocol-not-found-for-normal-file-name
